resources/spec.ejs

```diff
@@ -1,21 +1,28 @@
-<% if (name) { %>Name: <%= name %>
-<% } %><% if (version) { %>Version: <%= version %>
-<% } %><% if (revision) { %>Release: <%= revision %>%{?dist}
-<% } %><% if (description) { %>Summary: <%= description %>
-<% } %><% if (homepage) { %>URL: <%= homepage %>
-<% } %><% if (license) { %>License: <%= license %>
-<% } %>
-AutoReqProv: no
-<% if (requires) { %>Requires: <%= requires.join(', ') %>
-<% } %><% if (productDescription) { %>
-%description
-<%= productDescription %>
-<% } %>
 %define _binary_payload w<%= compressionLevel %>.xzdio
 
-%install
+Name: <%= name %>
+Version: <%= version %>
+Release: <%= revision %>%{?dist}
+<% if (description) { print(`Summary: ${description}\n`) }
+%>
```

Comment:
Any reason why this couldn't be on one line?
```suggestion
<% if (description) { print(`Summary: ${description}\n`) } %>
```

Reply:
It can, but I put the empty lines so the SPEC file looks exactly how it will output. In other words, there has to be empty line between `description` and the block that contains `license` and/or `homepage`